### PR TITLE
fix: missing tooltip in quickpanel

### PR DESCRIPTION
**Pull request by 18202781743**
create surface for global ToolTipWindow, and it's extent property
is from it's transientParent,
and then update it's position in next event loop.

### CODE REVIEW

File: src/loader/quickpluginitem.cpp [hunk 1]
```diff
@@ -134,9 +134,10 @@ QMenu *QuickPluginItem::pluginContextMenu()
 
 QWidget *QuickPluginItem::pluginTooltip()
 {
-    auto popup = pluginsItemInterface()->itemPopupApplet(m_itemKey);
-    if (popup && popup->isVisible())
-        popup->windowHandle()->hide();
+    return nullptr;
```
Comment (tsic404) [inline src/loader/quickpluginitem.cpp:137]: 这个应该都可以去掉吧

File: src/tray-wayland-integration/plugin.cpp [hunk 2]
```diff
@@ -280,12 +286,21 @@ void PluginPopup::setY(const int& y)
...
+    return nullptr;
+}
 PluginPopup* PluginPopup::get(QWindow* window)
 {
     auto popup = s_popupMap.value(window);
     if (!popup) {
         popup = new PluginPopup(window);
         s_popupMap.insert(window, popup);
+        QObject::connect(popup, &PluginPopup::destroyed, window, [window] () {
+            s_popupMap.remove(window);
```
Comment (tsic404) [inline src/tray-wayland-integration/plugin.cpp:302]: popup 是new 出来的，这儿没有释放吧

Comment (18202781743) [reply src/tray-wayland-integration/plugin.cpp:302]: 它的parent是window, 可以不用,

File: src/loader/quickpluginitem.cpp (hunk 1 above)
Comment (18202781743) [reply src/loader/quickpluginitem.cpp:137]: 先不去掉吧, pluginitem也是用的这个虚函数,

**Review by tsic404 — APPROVED** [review]

**Pull request merged**